src/main/java/core/basesyntax/service/CsvFileWriter.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.WriteFileException;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvFileWriter implements FilesWriter {
+    @Override
+    public void write(String data, String fileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(fileName))) {
+            bufferedWriter.write(data);
+        } catch (IOException e) {
+            throw new WriteFileException("Can't write data at file " + fileName);
```

Comment:
We can just throw new RuntimeException with custom message when handling JDK checked exceptions, there is no need for custom exception in this case

Reply:
if I'm not mistaken, this is bad practice, for our specific mistakes we must create own exception

Reply:
This is not our mistake, this is IOException))

Reply:
In this case it is possible, but in my opinion it is not informative